src/Helper/Dto/Catalog/Product/Attribute/Value/Create.php

```diff
@@ -57,18 +57,25 @@ public function __construct(Locale $locale, SwatchHelper $swatchHelper)
      */
     public function execute(MageAttribute $attribute, string $locale, int $storeId): array
     {
-        $return    = [];
-        $orderList = $attribute->getData('option')['order'] ?? [];
-        $valueList = $attribute->getData('option')['value'] ?? [];
+        $return = [];
+        $allOptions = $attribute->setstoreId($storeId)
```

Comment:
no such method, even if u capitalize. You get a warning right? Try to minimize warnings as they may mask other issues. Speaking of... see below.
```suggestion
        $allOptions = $attribute->setData('store_id', $storeId)
```